test/system/103-run.bats

```diff
@@ -19,6 +19,29 @@ teardown() {
   cd "$CURDIR" || return 1
 }
 
+@test "run: containerstart: Check container starts without issues" {
+
+  create_default_container
+
+  run $PODMAN start "$(get_system_id)-toolbox-$(get_system_version)"
```

Comment:
Maybe use a variable for the container name?

Reply:
Cool. Adding this to code :+1: